Sources/SwifterSwift/UIKit/UIButtonExtensions.swift

```diff
@@ -167,34 +167,72 @@ public extension UIButton {
         states.forEach { setTitle(title, for: $0) }
     }
 
-    /// SwifterSwift: Center align title text and image
+    /// SwifterSwift: Set the image to the left of the text
+    ///
     /// - Parameters:
-    ///   - imageAboveText: set true to make image above title text, default is false, image on left of text
     ///   - spacing: spacing between title text and image.
-    func centerTextAndImage(imageAboveText: Bool = false, spacing: CGFloat) {
-        if imageAboveText {
-            // https://stackoverflow.com/questions/2451223/#7199529
-            guard
-                let imageSize = imageView?.image?.size,
-                let text = titleLabel?.text,
-                let font = titleLabel?.font else { return }
-
-            let titleSize = text.size(withAttributes: [.font: font])
-
-            let titleOffset = -(imageSize.height + spacing)
-            titleEdgeInsets = UIEdgeInsets(top: 0.0, left: -imageSize.width, bottom: titleOffset, right: 0.0)
-
-            let imageOffset = -(titleSize.height + spacing)
-            imageEdgeInsets = UIEdgeInsets(top: imageOffset, left: 0.0, bottom: 0.0, right: -titleSize.width)
-
-            let edgeOffset = abs(titleSize.height - imageSize.height) / 2.0
-            contentEdgeInsets = UIEdgeInsets(top: edgeOffset, left: 0.0, bottom: edgeOffset, right: 0.0)
-        } else {
-            let insetAmount = spacing / 2
-            imageEdgeInsets = UIEdgeInsets(top: 0, left: -insetAmount, bottom: 0, right: insetAmount)
-            titleEdgeInsets = UIEdgeInsets(top: 0, left: insetAmount, bottom: 0, right: -insetAmount)
-            contentEdgeInsets = UIEdgeInsets(top: 0, left: insetAmount, bottom: 0, right: insetAmount)
-        }
+    func setImageLeftOfText(spacing: CGFloat) {
+        let insetAmount = spacing / 2
+        imageEdgeInsets = UIEdgeInsets(top: 0, left: -insetAmount, bottom: 0, right: insetAmount)
+        titleEdgeInsets = UIEdgeInsets(top: 0, left: insetAmount, bottom: 0, right: -insetAmount)
+    }
+
+    /// SwifterSwift: Set the image to the right of the text
+    ///
+    /// - Parameters:
+    ///   - spacing: spacing between title text and image.
+    func setImageRightOfText(spacing: CGFloat) {
+        // https://stackoverflow.com/questions/2451223/#7199529
+        guard
+            let imageSize = imageView?.image?.size,
+            let text = titleLabel?.text,
+            let font = titleLabel?.font else { return }
+
+        let titleSize = text.size(withAttributes: [.font: font])
+
+        let titleOffset = -(imageSize.width + spacing / 2)
+        titleEdgeInsets = UIEdgeInsets(top: 0, left: titleOffset, bottom: 0, right: -titleOffset)
+
+        let imageOffset = -(titleSize.width + spacing / 2)
+        imageEdgeInsets = UIEdgeInsets(top: 0, left: -imageOffset, bottom: 0, right: imageOffset)
+    }
```

Comment:
```suggestion
    func setImageRightOfText(spacing: CGFloat) {
        guard
            let imageSize = imageView?.image?.size,
            let text = titleLabel?.text,
            let font = titleLabel?.font else { return }

        let inset = spacing / 2

        let titleSize = text.size(withAttributes: [.font: font])

        let titleOffset = -(imageSize.width + inset)
        titleEdgeInsets = UIEdgeInsets(top: 0, left: titleOffset, bottom: 0, right: -titleOffset)

        let imageOffset = -(titleSize.width + inset)
        imageEdgeInsets = UIEdgeInsets(top: 0, left: -imageOffset, bottom: 0, right: imageOffset)

        contentEdgeInsets = UIEdgeInsets(top: 0, left: inset, bottom: 0, right: inset)

        sizeToFit()
    }
```

Reply:
This is strange. I tested with your code and found that as long as the spacing is too large, the image will exceed the width of the button and be cut off. Then I found a way to meet the same style as `setImageLeftOfText`.
Your:
<img width="287" alt="your" src="https://user-images.githubusercontent.com/12928482/98517450-e4003100-22a8-11eb-9000-08b0bcfcf322.png">

new:
<img width="282" alt="new" src="https://user-images.githubusercontent.com/12928482/98517491-f4181080-22a8-11eb-9197-a5e4b95ff558.png">

```
    func setImageRightOfText(spacing: CGFloat) {
        semanticContentAttribute = .forceRightToLeft
        
        let insetAmount = spacing / 2
        imageEdgeInsets = UIEdgeInsets(top: 0, left: insetAmount, bottom: 0, right: -insetAmount)
        titleEdgeInsets = UIEdgeInsets(top: 0, left: -insetAmount, bottom: 0, right: insetAmount)
        contentEdgeInsets = UIEdgeInsets(top: 0, left: insetAmount, bottom: 0, right: insetAmount)
    }
```
What do you think of this way?

Reply:
No, I don't like that hack.

Reply:
As I said above, if you don’t use this method, no matter how you set `contentEdgeInsets`, when the spacing is too large, the image will always exceed the bounds of the button.